### PR TITLE
change to new CCF container registry

### PR DESCRIPTION
**Pull request by beejones**
Updated the docker container to the latest 5.0.4 CCF image.
After running dependabot we have several packages that cannot be installed automatically. This is fixed in this PR.

### CODE REVIEW

File: docker/Dockerfile [hunk 1]
```diff
@@ -49,4 +49,4 @@ WORKDIR /kms
 
 ARG CCF_PLATFORM
 ENV CCF_PLATFORM=${CCF_PLATFORM}
-CMD ["/bin/bash", "-c", "make start-host-idp & (./scripts/kms_wait.sh && make setup && tail -f /dev/null)"]
+CMD ["/bin/bash", "-c", "source .venv_ccf_sandbox/bin/activate && make start-host-idp & (./scripts/kms_wait.sh && make setup && tail -f /dev/null)"]
```
Comment (DomAyre) [inline docker/Dockerfile:52]: Why are we doing this?

`make start-host-idp` calls `make build` which calls `./scripts/set_python_env.sh` which activates the virtual environment

Comment (DomAyre) [reply docker/Dockerfile]: Resolved by merging main in

**Comment by DomAyre** [comment]
Can you provide a link to a privacy-sandbox-dev CI run which shows this doesn't break anything? Otherwise looks okay to me

**Comment by beejones** [comment]
> Can you provide a link to a privacy-sandbox-dev CI run which shows this doesn't break anything? Otherwise looks okay to me

https://github.com/microsoft/privacy-sandbox-dev/actions/runs/10698226659

**Pull request merged**